ui/intermittent-failures/View.jsx

```diff
@@ -106,18 +101,42 @@ const withView = defaultState => WrappedComponent =>
     this.setState({ graphData: data, graphFailureStatus: failureStatus, isFetchingGraphs: false });
   }
 
-  updateState(updatedObj, updateTable = false) {
+  async batchBugRequests(bugIds) {
+    const urlParams = {
+      include_fields: 'id,status,summary,whiteboard',
+    };
+    // TODO: bump up the max to ~1200 when this bug is fixed:
+    // https://bugzilla.mozilla.org/show_bug.cgi?id=1497721
+    let min = 0;
+    let max = 800;
+    let bugsList = [];
+    const results = [];
+
+    while (bugIds.length >= min) {
+      const batch = bugIds.slice(min, max + 1);
+      urlParams.id = batch.join();
+      results.push(getData(bugzillaBugsApi('bug', urlParams)));
+
+      min = max;
+      max += 800;
+    }
+
+    for (const result of await Promise.all(results)) {
+      bugsList = [...bugsList, ...result.data.bugs];
+    }
+    return bugsList;
+  }
+
+  updateState(updatedObj) {
     this.setState(updatedObj, () => {
-      const { startday, endday, tree, page, pageSize, bug } = this.state;
-      const params = { startday, endday, tree, page, page_size: pageSize };
+      const { startday, endday, tree, bug } = this.state;
+      const params = { startday, endday, tree };
 
```

Comment:
Since the pagination was removed in the endpoints, these are no longer needed or applicable.